libs/ui-watt/src/lib/foundations/typography/+storybook/shared/typography-html-snippets.ts

```diff
@@ -22,46 +22,57 @@ function mixinSnippet(name: string) {
 
 export const typographyHtmlSnippets = {
   h1: {
-    tag: '<h1>Headline 1</h1>',
-    class: '<p class="watt-headline-1">Headline 1</p>',
+    tag: '<h1>headline 1</h1>',
+    class: '<p class="watt-headline-1">headline 1</p>',
```

Comment:
nitpick: Some of the text values start with a capital letter, others not. Some have a dash in between, other don't. Sync with Kenneth to normalize the text.

Reply:
should be fixed now